### PR TITLE
fix format error

### PR DESCRIPTION
**Pull request by xieyanker**
image

### CODE REVIEW

File: content/zh/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -2418,4 +2418,5 @@ sudo systemctl restart docker
 -->
 
 * 参考[使用持久卷部署 WordPress 和 MySQL](/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/) 示例。
+
```
Comment (tanjunchen) [inline content/zh/docs/concepts/storage/volumes.md:2421]: Thanks for your job!
/lgtm

**Comment by tengqm** [comment]
/approve
Not sure it fixes all issues, but this is harmless at least.

**Comment by xieyanker** [comment]
@tengqm Sorry, it seems no help. Is there any idea to fix this?
https://kubernetes.io/zh/docs/concepts/storage/volumes/

**Pull request merged**